src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -267,6 +269,13 @@ public void run() {
 
     if (newVar != null) {
       if (intoScalar) {
+        Preconditions.checkArgument(
+            type.isRepeated() && !type.isMap(),
```

Comment:
Don't we want to allow iterating over maps?

Reply:
I think we should hold off on this. It's unclear what you're iterating over. Keys? Values? Pairs?

Reply:
Agreed, but we need to decide this before we declare the loop construct to be fully implemented.....